pysb/core.py

```diff
@@ -359,11 +369,13 @@ def __init__(self, monomer, site_conditions, compartment):
                  all(isinstance(s, int) for s in state):
                 continue
             elif isinstance(state, basestring):
+                self.__class__._check_state(monomer, site, state)
```

Comment:
This doesn't really read well as a staticmethod. Can it just be a top-level underscore-prefixed function?